doc_template/index.rst

```diff
@@ -25,6 +25,9 @@ Welcome to the Allen SDK
 The Allen Software Development Kit houses source code for reading and processing Allen Brain Atlas data.  
 The Allen SDK focuses on the Allen Brain Observatory, Cell Types Database, and Mouse Brain Connectivity Atlas.
 
+.. ATTENTION::
+    We will be dropping for py2 support in October 2019
```

Comment:
add a message that files with a py2 dependency (e.g. analysis files) will also be updated.